module/trace/log_tracer.go

```diff
@@ -62,6 +62,11 @@ func (t *LogTracer) Start(
 	return ctx, sp
 }
 
```

Comment:
maybe we should just remove log tracer-- i haven't ever seen anyone use it and it's just slowly rotting away.

Reply:
sure.  makes my life easier